lib/docs/src/tag.stories.tsx

```diff
@@ -1,61 +1,56 @@
 import { Meta } from "@storybook/react";
-import { Fragment } from "react";
+import { Fragment, useState } from "react";
 import { Button, DivPx, Tag } from "../../core/src";
 import { Utils } from "./utils";
-
-const colors: string[] = [
-	"red",
-	"yellow",
-	"green",
-	"blue",
-	"indigo",
-	"purple",
-	"pink",
-	"gray",
-];
+import { GalleryFeedbackTag } from "../../gallery/src/feedback/tag";
 
 const meta: Meta = {
 	title: "Components/Tag",
 	component: Tag,
 	argTypes: {
 		children: Utils.arg("text"),
-		color: Utils.arg(colors),
+		color: Utils.arg(Object.keys(Tag.colors)),
 	},
 };
 
-Utils.page.component(meta, { sticky: true, shots: [] });
+Utils.page.component(meta, { sticky: true, shots: [<GalleryFeedbackTag />] });
 
 export default meta;
 
 interface Props {
-	children?: string;
-	color?: string;
+	children: string;
+	color: string;
```

Comment:
These are from Storybook's arg table controllers, so I think it's expected that they might be undefined. In other words, "color?: string" is correct, not "color: string"